core/templates/dev/head/pages/exploration_editor/translation_tab/translator_overview_directive.html

```diff
@@ -47,4 +47,29 @@
     padding: 2px;
     border-radius: 8px;
   }
+  /* Firefox rules */
+  @-moz-document url-prefix(){
```

Comment:
Style: add space before {

/cc @apb7

Reply:
This has already been added to #5097. Thanks!